mailu/Chart.yaml

```diff
@@ -2,4 +2,5 @@ apiVersion: v1
 appVersion: "1.8"
 description: Mailu mail system
 name: mailu
+icon: https://mailu.io/1.7/_images/logo.png
```

Comment:
Not related but I'll accept it if you use https://mailu.io/master/_images/logo.png ;)